### PR TITLE
feat(alerts): Add new option to allow members to edit alerts

### PR DESCRIPTION
**Pull request by davidenwang**
Adds a new setting in the organization general settings "Grant Members Alerts Write":

image

This will enable members to edit alerts by giving them the `alerts:write` permission (added in https://github.com/getsentry/sentry/pull/22800)

The behavior of this option is that old organizations will have it set to False (through https://github.com/getsentry/sentry/pull/22852) as to not disrupt their current permissions while new orgs will follow the default of having this set to True so that members can create/edit/delete alerts

### CODE REVIEW

**Review by sydjryan — COMMENTED** [review]
this is fine from data team's perspective

**Review by wedamija — COMMENTED** [review]
This looks good, but we probably need a test around `get_scopes`

File: tests/sentry/models/test_organizationmember.py [hunk 1]
```diff
@@ -202,3 +202,19 @@ def test_scopes_with_member_admin_config(self):
...
+
+        assert "alerts:write" in member.get_scopes()
+
+        organization.update_option("sentry:alerts_member_write", True)
+
+        assert "alerts:write" in member.get_scopes()
+
+        organization.update_option("sentry:alerts_member_write", False)
+
+        assert "alerts:write" not in member.get_scopes()
```
Comment (wedamija) [inline tests/sentry/models/test_organizationmember.py:220]: Just one extra part to this test: Update the member to be an admin and verify that the setting doesn't break the scope for them.

**Review by wedamija — APPROVED** [review]

File: tests/sentry/models/test_organizationmember.py (hunk 1 above)
Comment (davidenwang) [reply tests/sentry/models/test_organizationmember.py:220]: oooh shoot yea good catch

**Pull request merged**